llvm/test/CodeGen/AMDGPU/wave32-to-64-auto-convert.ll

```diff
@@ -0,0 +1,121 @@
+; RUN: opt -S -mtriple=amdgcn -mcpu=gfx1100 -passes=si-convert-wave-size < %s | FileCheck %s
```

Comment:
Also need to test full compilation and check resulting .amdhsa_wavefront_size32 value.